fusor-ember-cli/app/templates/components/rhci-start.hbs

```diff
@@ -43,7 +43,7 @@
   </div>
 
   <div class='download-reqs'>
-    <a download href="QCI_Requirements.txt">
+    <a href="/assets/r/QCI_Requirements.txt" target="_blank">
```

Comment:
@isratrade the link path was not correct. We're you testing with ember dev server? That will serve static from a different location than where it needs to be for Foreman. I will check this morning and see if I can add in the download, sounds like we should keep that behavior.